XRTK-Core/Packages/com.xrtk.core/Editor/PropertyDrawers/PlatformEntryPropertyDrawer.cs

```diff
@@ -313,7 +342,14 @@ void OnEditorSelected(object _)
             void OnSelectedTypeName(object typeRef)
             {
                 var selectedPlatformType = typeRef as Type;
-                var selectedReference = SystemType.GetReference(selectedPlatformType);
+
+                if (selectedPlatformType != null && selectedPlatformType.GUID == Guid.Empty)
+                {
+                    Debug.LogError($"{selectedPlatformType.Name} does not implement a required {nameof(GuidAttribute)}");
+                    return;
+                }
+
+                var selectedReference = selectedPlatformType?.GUID.ToString();
```

Comment:
Can we not make all these references cleaner by having the respective functions simply accept a GUID, then only use "toString" in side the function if needed.  Removing the need for all these "ToString" function calls everywhere.

Reply:
No, because we'd break backwards compatibility with `AssemblyQualifiedName`

Reply:
Well do the toString cast inside the function, rather than everywhere that calls the function at the moment, cleaner

Reply:
Some places we're using the serialized property (which may still have the old `AssemblyQualifiedName` serialization.

Reply:
Well, once you've created the overload, those places that can should use the new overload, save unnecessary conversion